### DISCUSSION TITLE
MacOS Uninstall / Reset

### DISCUSSION THREAD

**Discussion by rglasgow-maxpreps** [zed-industries/zed#6996] [Feature Requests]
How do I uninstall Zed completely? Ideally, it takes me back through the new account creation tool when I re-install the application.

**Why?**
My LSP for TSX and TS are not working even after multiple attempts to get it up and running. 

***Edit***
After trying several languages, GO, TSX, TS, RUST, Python, etc... None of the LSPs are working for them.

**Comment by sihayas** [comment]
did you figure this out

**Reply by rglasgow-maxpreps** [reply]
Yes, the solution is to go to configs and global installs of the LSPs (specific to the machine) and remove them. Zed will automatically download them if they are available. Mine were in the user's home directory. You may need to show the hidden files.
